spec/controllers/guides_controller_spec.rb

```diff
@@ -67,4 +67,12 @@
       end
     end
   end
+
+  describe "a malicious user trying to initialise an unwanted Guide STI constant" do
```

Comment:
Yeah - somewhere that `params[:type]` string will need to be converted into a constant for the STI to do it's thing.

```
irb(main):003:0> Guide.new(type: 'Point')
=> #<Point id: nil, slug: nil, content_id: nil, created_at: nil, updated_at: nil, tsv: nil, type: "Point">
```

Rails probably has some kind of protection around it.. but I just don't know what it is. It'll error if you try and initialise a class that isn't a subclass of Guide. But what else could you try and throw in there? Scary.